static/scripts/actions.jsx

```diff
@@ -571,7 +616,9 @@ export function fetchPredictions() {
       dispatch,
       FETCH_PREDICTIONS,
 
-      fetch('/predictions')
+      fetch('/predictions', {
+        credentials: 'same-origin'
```

Comment:
Perhaps this should be the default param, which would clean these calls up a bit.